modules/javafx.graphics/src/test/java/test/javafx/stage/WindowTest.java

```diff
@@ -82,7 +82,8 @@ public void testOpacityBind() {
         assertEquals(1.0f, peer.opacity);
     }
 
-    @Test public void testProperties() {
+    @Test
```

Comment:
Minor - I think it would be best to keep old code untouched to not mix up the file history in Git, should we ever need it.

Reply:
Good suggestion. The following sentence from the [code style section of the CONTRIBUTING guidelines](https://github.com/openjdk/jfx/blob/master/CONTRIBUTING.md#coding-style-and-testing-guidelines) provides this advice:

* Avoid making changes that are unrelated to the bug you are fixing. This includes fixing minor errors such as warnings, spacing / indentation, spelling errors, etc, in code that you aren't otherwise modifying as part of your fix.

Reply:
Done.